### DISCUSSION TITLE
Rider and Uno I seem to be missing something ....

### DISCUSSION THREAD

**Discussion by stephenhauck** [unoplatform/uno#15209] [General]
So I did all the setup and updated to the latest Rider IDE today on Mac ..
I created a new project and I have no "run / debug" configurations and such ...
Are the "wizards" they show in the videos in Rider or only in Visual Studio ?

**Comment by limeniye** [comment]
Hi, @stephenhauck !
Please, check the following discussion: https://github.com/unoplatform/uno/discussions/13335#discussioncomment-6784649
And read this docs: 
https://platform.uno/docs/articles/get-started-rider.html?tabs=windows%2Cubuntu1804
https://platform.uno/docs/articles/create-an-app-rider.html?tabs=android

**Comment by stephenhauck** [comment]
@limeniye thanks for that but the screenshots don't  look like the current solution structure in Rider 
Also there was no "wizard when creating a new Uno app ...
<img width="329" alt="image" src="https://github.com/unoplatform/uno/assets/12648793/a68575df-3bb1-41b7-b155-29548d069b3c">

**Reply by jeromelaban** [reply]
We currently have no mention of a Wizard in our Rider docs, and if there is, it's an error. Rider is missing a feature for us to include an embedded wizard-like experience.

**Reply by stephenhauck** [reply]
That's fine but still how do I actually run an Uno app ?

**Reply by jeromelaban** [reply]
If you did use `dotnet new` template, you should be getting the solution in the same way as in the docs. What command did you use to create your project?

**Reply by stephenhauck** [reply]
Rider - new solution uno app

**Reply by jeromelaban** [reply]
I see, thanks. We do not recommend using the Rider new solution app. Try [using the command line instead](https://platform.uno/docs/articles/create-an-app-rider.html?tabs=android#create-the-app).

**Reply by stephenhauck** [reply]
Thanks for your help but honestly with Visual Studio on Mac being deprecated and Uno not seeming to truly be supported by Rider I think I will simply pass on Uno all together and simply stick with buggy MAUI and Flutter .....

I was hoping Uno could replace both MAUI and Flutter ..

Will check it out in another year maybe ...

**Reply by jeromelaban** [reply]
Uno's templates are rich, and the missing features from rider prevent us from building something that makes sense. You can [upvote this](https://youtrack.jetbrains.com/issue/RIDER-16759), that would help. Still, as rider runs `dotnet new` without parameters, it should still be working as the docs says and it may be a recent rider regression.

That being said, if you're willing to give a try to running a simple command line to get past this Rider limitation, you may be able to get further along and actually try what Uno has to offer.

**Reply by stephenhauck** [reply]
Again .. thanks .. Uno looks excellent but I deal with enough "fragmented toolchains" with Flutter every day ... 
I just don't have the time to fool with it ... upvoting won't help because if everyone upvoted it still has to be scheduled and the work done ... 

Thanks

**Reply by jeromelaban** [reply]
As a follow-up those are the Rider issues that we're tracking: https://github.com/unoplatform/uno/issues/15226, including the one that contains the missing projects that you've noticed.

**Comment by stephenhauck** [comment]
<img width="465" alt="image" src="https://github.com/unoplatform/uno/assets/12648793/2892a18b-090b-4986-adc1-65c8eab5497b">

**Reply by limeniye** [reply]
Good news for you.
The Uno Platform team is preparing a new release with a Live Wizard option.
Let's read it there:
https://platform.uno/blog/uno-platform-5-1/

**Reply by stephenhauck** [reply]
Interesting .. now I need to figure out how to setup Uno with Codemagic for my CI / CD

**Comment by stephenhauck** [comment]
So I found the live wizaed and used it in VS Code on my mac ...
https://new.platform.uno/WizardContentHost/Welcome

Then opened it with Rider ... seems to work well so far
